woocommerce/woocommerce-ios
# [Crash] Update podfile to use fixed MediaPicker-iOS

Closes: #11838 


## Description


Updating to the latest version of MediaPicker-iOS should get us various fixes for the index out of range crash above.

We couldn't do that directly to 1.8.10, because it broke the photo taking flow if a limited library selection permission is applied. https://github.com/wordpress-mobile/MediaPicker-iOS/issues/419

This PR updates the app to use a fixed version of MediaPicker-iOS, pending the PR review there. https://github.com/wordpress-mobile/MediaPicker-iOS/pull/420

## Testing instructions


1. Launch the app
2. Tap Products
3. Select a product
4. Tap the add media cell in the photos row
5. Tap `Add Photo`
6. Tap `Take a Photo`
7. Take a photo, and tap `Use Photo`
8. Observe that the photo is added to the product, and no error is shown.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/92770203-8696-4875-a7f7-593e1b25c760



---
